repose-aggregator/components/filters/translation/pom.xml

```diff
@@ -87,11 +86,6 @@
             <artifactId>xalan</artifactId>
             <version>2.7.1</version>
         </dependency>
-        <dependency>
```

Comment:
for the record from a parent pom is different, because that is effectively a direct dependency. but a dependency of our dependency is not kosher